Doc/using/configure.rst

```diff
@@ -185,6 +185,15 @@ General Options
 
    .. versionadded:: 3.11
 
+.. cmdoption:: --disable-gil
+
+   Enables experimental support for running Python without the
```

Comment:
@AA-Turner: Maybe just this emphasis? :-)

```suggestion
   Enables **experimental** support for running Python without the
```

In the past, I added an **undocumented** ``--experimental-isolated-subinterpreters`` option which has been removed in the meanwhile: https://github.com/python/cpython/issues/84694 Well, https://peps.python.org/pep-0684/ is now accepted, no option is needed anymore :-)

Reply:
I think this is good, thanks Victor